addons/web_editor/models/ir_attachment.py

```diff
@@ -29,8 +29,13 @@ def _compute_image_src(self):
             if attachment.mimetype not in ['image/gif', 'image/jpe', 'image/jpeg', 'image/jpg', 'image/gif', 'image/png', 'image/svg+xml']:
                 attachment.image_src = False
             else:
-                attachment.image_src = attachment.url or '/web/image/%s/%s' % (
+                # Adding unique to url as cachebuster, currently doesn't leverage Max-age headers.
+                attachment.image_src = '%s?unique=%s' % (
+                    attachment.url,
+                    attachment.checksum,
+                ) if attachment.url else '/web/image/%s-%s/%s' % (
                     attachment.id,
+                    attachment.checksum[0:8],
```

Comment:
```suggestion
        # Adding unique to url as cachebuster, currently doesn't leverage Max-age headers for now.
        unique = attachment.checksum[:8]
        if attachment.url:
            attachment.image_src = '%s?unique=%s' % (attachment.url, unique)
        else:
            attachment.image_src = '/web/image/%s-%s/%s' % (attachment.id, unique),
```

Reply:
don't try to obfuscate your code ;)

Reply:
@JKE-be I pushed a commit on top of the branch to solve that (I was planning to squash it and merge) but in fact I think it is wrong here: we cannot just append ?unique= at the end of every attachment with an URL (they might be a URL-type attachment whose URL is an external one).
So three solutions to me:
- I leave `attachment.image_src = attachment.url` for url-type attachment and just append ?unique= for binary type attachment which have an URL (supposing there is no query string in those we defined, or should I parse the query string?)
- I leave attachment.image_src = attachment.url here and in unsplash I make an override to append the ?unique=, only case I know where we actually need the unique for now.
- I parse the query string of the URL here and add the unique in all cases, praying that "unique" parameter as no meaning in the external url

I'd go with the first one but I prefer to have your opinion (and if I should parse the query string for robustness or not).

Reply:
@JKE-be In fact, checksum does not always exist so this could not work at all. I added a new commit on top to check for the existence of checksum... but we should probably check why it is possible?

cc @sdegueldre I also rebased the branch and push-forced it (with the rebase alone).